src/copy.rs

```
@@ -170,6 +170,12 @@ pub enum SourceCreationError {
 
     #[error("Couldn't truncate the temporary file for newline trimming")]
     TempFileTruncate(#[source] io::Error),
+
+    #[error("Couldn't create the memfd file")]
+    MemFdFileCreate,
+
+    #[error("Couldn't seal the memfd file")]
+    MemFdFileSeal,
```

these should be cfg'd as well, since these errors are never returned when the feature is off.